lib/consts.tsx

```diff
@@ -32,9 +32,13 @@ export const EVENT_ZORA_PROFILE = 'zora+profile';
 // zora score
 export const FOLLOWERS_PERFECT = 55555;
 export const MAX_CREATE_SCORE = 111;
-export const SCORE_FACTOR = 0.5;
+export const SCORE_FACTOR = 0.33;
+export const MAX_REWARD_SCORE = 0.222;
 
 // stack point systems
 export const TOKEN_INDEXER_POINT_ID = 3500;
 export const REWARDS_DEPOSIT_POINT_SYSTEM_ID = 4272;
 export const SETTINGS_STACK_ID = 3067;
+
+// Conversions
+export const ETH_TO_WEI = Math.pow(10, 18);
```

Comment:
Please use `viem` to convert `wei`<=> `eth`
- example lib: `formatEther`
https://viem.sh/docs/utilities/formatEther#formatether